package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "exec-buffer",
-  "version": "3.0.0",
+  "version": "3.0.1",
```

Comment:
If you're going to do that you might as well update it, I'll just close this then.